public/locales/en/gallery.json

```diff
@@ -1 +1,7 @@
-{}
+{
+    "not applicable": "not applicable",
```

Comment:
Can you also remove the spaces from the keys here? We really want to avoid that. The tags are a special case since we are looking up using the actual tag name, but otherwise we shouldn't include spaces in the keys.

Reply:
yup